### ISSUE TITLE
Listening to wrong event

### ISSUE THREAD

**Issue by ellimondo** [edalzell/statamic-mailchimp#36]
Hi

Is there anywhere I can see if the add on is communication/sending out anything to MailChimp?

I get no response from MailChimp upon form submission and no error in the log files so I am at a bit of a loss on how to proceed.

The form is submitting fine to Statamic through both Postmark API and log mailer settings. I am working locally.

**Comment by edalzell** [comment]
Let’s see your mailchimp settings

**Comment by ellimondo** [comment]
```
<?php

return [

    'add_new_users' => false,

    'forms' => [
        [
            'audience_id' => 'IDHERE',
            'check_consent' => false,
            'consent_field' => 'consent',
            'form' => 'contact',
            'merge_fields' => [
                [
                    'tag'=> 'EMAIL',
                    'field_name' => 'email'
                ],
                [
                    'tag'=> 'FNAME',
                    'field_name' => 'first_name'
                ],
                [
                    'tag'=> 'LNAME',
                    'field_name' => 'surname'
                ]
            ],
            'disable_opt_in' => false,
        ],
    ],
];
```
The error has disappeared in the last update thanks.

BTW Your readme notes still have the missing '[]' in the example.

I've tried various basic forms, some with just email in the Merge array. All just silently fail. But there's no response anywhere. Laravel log remains empty.

Sorry to be a pain.

BTW I am using PHP 7.4.7 - I wonder if the MailChimp API wrapper is broken in this version?

**Comment by edalzell** [comment]
Issue confirmed @ellimondo, working on a fix.

**Comment by edalzell** [comment]
Sorry, got confused, still trying to figure out what's wrong.

**Comment by edalzell** [comment]
Fixed in 2.0.3 @ellimondo

**Comment by ellimondo** [comment]
Parp! That works. Thanks so much. Jon.
